keyboards/lily58/keymaps/mikefightsbears/keymap.c

```diff
@@ -0,0 +1,237 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
```

Comment:
You only have one base layer and aren't using the `QWERTY` keycode. You can modify this enum and remove the `QWERTY` keycode logic for no loss of functionality by changing Lines 26-27 to:

```
  LOWER = SAFE_RANGE,
```